### DISCUSSION TITLE
Milvus2.3.0 GPU how to set GPU memory size.

### DISCUSSION THREAD

**Discussion by mydreambei-ai** [milvus-io/milvus#26760] [Q&A and General discussion] [answered]
https://milvus.io/docs/install_standalone-gpu-docker.md
the guide show use KNOWHERE_GPU_MEM_POOL_SIZE env but i set KNOWHERE_GPU_MEM_POOL_SIZE: 2G;12G does not effect .

**Comment by yhmo** [comment, marked as answer]
Should be `KNOWHERE_GPU_MEM_POOL_SIZE: 2048;12288`

We will update the doc later.
